src/stories/theme/_list.css

```diff
@@ -0,0 +1,33 @@
+.ck-content ul {
+  margin: 16px 0px 16px 20px;
+  padding: 0;
+  list-style-position: outside;
+  list-style-type: disc;
```

Comment:
Ideally, there would be styling that corresponds with the nested form seen under https://garden.zendesk.com/components/lists#nesting. Following Google docs, this is `1. - a. - i.` (repeating) for ordered and `disc - circle - square` (repeating) for unordered.

Reply:
We are unable to enforce nested list sequencing as that can be controlled via a separate plugin that allows consumers to choose them directly. [Here is an example of those dropdowns in action.](https://ckeditor.com/ckeditor-5/demo/#document-paged)

This is also business logic, correct? Guide currently enforces these values from what is in the Copenhagen theme, which doesn't align with our values.

Reply:
In the case where ordering cannot be automated, it is still important to capture and expose the Garden opinion on this matter. In this case, there should be a plan for some kind of "implementation notes" documentation that points to the plugin and sets expectations for the default ordering to be used within Zendesk. Whether that is exposed within the repo's README, within internal design/dev documentation, or some other form is a topic for team conversation that should be fleshed out and determined.

Reply:
For sure. I'm taking some additional notes in the audit doc so we can return to this.